snapcraft.yaml

```diff
@@ -941,6 +941,11 @@ parts:
       - --prefix=/usr
       - --with-builtin=pulse
     build-environment: *buildenv
+    override-build: |
+      set -eux
+      cd $CRAFT_PART_BUILD
+      ./autogen.sh --prefix=/usr
```

Comment:
is there a difference on running this with options versus running it sans options and then running the configure script with this option?

Reply:
Thanks for reviewing.

--prefix=/usr has no effect. autogen.sh only runs configure if the first argument is "c". Otherwise the argument is ignored. So I should drop it here.